### DISCUSSION TITLE
Code for Benchmarks?

### DISCUSSION THREAD

**Discussion by sno2** [pest-parser/pest#623] [General]
Hi, is there anywhere we can find the code for the benchmarks on the website?

![image](https://user-images.githubusercontent.com/43641633/124158888-193bd800-da60-11eb-8b08-de71a1057356.png)

**Comment by tomtau** [comment]
I'm not sure of the origin.
In any case, there's one older benchmark: https://github.com/Geal/pestvsnom
and this old write-up: https://unhandledexpression.com/general/2018/10/04/no-pest-is-not-faster-than-nom.html
